assets/js/common/CheckableWarningMessage/CheckableWarningMessage.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Input from '@common/Input';
+import { noop } from 'lodash';
+import { EOS_WARNING_OUTLINED } from 'eos-icons-react';
+
+function CheckableWarningMessage({
+  hideCheckbox = false,
+  warningText,
+  checked,
+  setChecked = noop,
+}) {
+  return (
+    <div className="flex items-center border border-yellow-400 bg-yellow-50 p-4 rounded-md text-yellow-600 mb-4">
+      {!hideCheckbox && (
+        <Input
+          type="checkbox"
+          checked={checked}
+          onChange={() => setChecked((prev) => !prev)}
+        />
+      )}
+      <EOS_WARNING_OUTLINED
+        size="xxl"
```

Comment:
We could add more props to have more control over the component, but not sure if we need it right away to be honest. 
What do you think ?